2016-11-13 16:38 DEBUG    manager                       Figuring out config load paths
2016-11-13 16:38 DEBUG    manager                       Found config: /Users/eivindml/.flexget/config.yml
2016-11-13 16:38 DEBUG    manager                       Config file /Users/eivindml/.flexget/config.yml selected
2016-11-13 16:39 CRITICAL plugin        ezrss.it        Unable to download the RSS for task ezrss.it (https://eztv.ag/ezrss.xml): HTTPSConnectionPool(host='eztv.ag', port=443): Max retries exceeded with url: /ezrss.xml (Caused by NewConnectionError('<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x11229f6d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
2016-11-13 16:39 WARNING  task          ezrss.it        Aborting task (plugin: rss)
2016-11-13 16:40 WARNING  requests.packages.urllib3.connectionpool                 Retrying (Retry(total=0, connect=None, read=None, redirect=0)) after connection broken by 'NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x1122cb210>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',)': /rss.php?feed=dl&passkey=74bc1121a91304aafb129c9dde9d8e95&format=bb
2016-11-13 16:40 INFO     download      norbits.net     Downloading: Modern.Family.S08E03.NORDiC.720p.WEB-DL.H.264-RAPiDCOWS
2016-11-13 16:40 INFO     transmission  norbits.net     "Modern.Family.S08E03.NORDiC.720p.WEB-DL.H.264-RAPiDCOWS" torrent added to transmission
2016-11-13 16:40 CRITICAL task          norbits.net     BUG: Unhandled error in plugin make_html: [Errno 2] No such file or directory: u'/Users/eivindml/public_html/series.html'
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/task.py", line 438, in __run_plugin
    return method(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/event.py", line 22, in __call__
    return self.func(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/FlexGet-1.2.495-py2.7.egg/flexget/plugins/output/html.py", line 41, in on_task_output
    with open(output, 'w') as f:
IOError: [Errno 2] No such file or directory: u'/Users/eivindml/public_html/series.html'
